server/app/views/questiontypes/RadioButtonQuestionRenderer.java

```diff
@@ -94,18 +94,25 @@ private DivTag renderRadioOption(
     LabelTag labelTag =
         label()
             .withFor(id)
-            .withClasses("inline-block", "w-full", "h-full")
-            .with(inputTag)
-            .with(span(option.optionText()).withClasses(ReferenceClasses.MULTI_OPTION_VALUE));
+            .withClasses(
+                ReferenceClasses.RADIO_OPTION,
+                BaseStyles.RADIO_LABEL,
+                checked ? BaseStyles.BORDER_CIVIFORM_BLUE : "",
+                "inline-block",
+                "w-full",
+                "h-full")
+            .with(
+                inputTag,
+                div()
+                    .with(
+                        TextFormatter.formatText(
+                            option.optionText(),
+                            /* preserveEmptyLines= */ true,
```

Comment:
`preserveEmptyLines=true` doesn't actually do anything right now since you can't enter new line characters in an input box.